albyhub/umbrel-app.yml

```diff
@@ -0,0 +1,29 @@
+manifestVersion: 1
+id: albyhub
+name: Alby Hub
+tagline: Your Hub to connect to the Bitcoin world
+category: finance
+version: "1.6.0"
+port: 59000
+description: >-
+  Alby Hub is your gateway to Bitcoin world and allows you to use your Umbrel lightning node with the growing number of exciting bitcoin and nostr applications.
+  Connect any bitcoin app to your Hub for self-custodial payments anywhere — websites, mobile apps, desktop apps, games, CLI tools and more.
+
+  Use Alby Hub to manage your lightning channels through one sleek, user-friendly interface.
```

Comment:
To get whitespace between the two paragraphs we need to add two newlines:
```suggestion
description: >-
  Alby Hub is your gateway to Bitcoin world and allows you to use your Umbrel lightning node with the growing number of exciting bitcoin and nostr applications.
  Connect any bitcoin app to your Hub for self-custodial payments anywhere — websites, mobile apps, desktop apps, games, CLI tools and more.


  Use Alby Hub to manage your lightning channels through one sleek, user-friendly interface.
```

This will now render nicely like this:

<img width="659" alt="image" src="https://github.com/user-attachments/assets/18e03d2c-db4b-4e5e-af5e-40e7338a24d5">